src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -280,23 +287,26 @@ private boolean isLetterHolderRole(String role) {
         }
 
         User letterHolder = userService.getUser(authorisation);
-        return Optional.of(readCase(letterHolder, letterHolderCases.get(0)));
+        CaseDetails caseDetails = readCase(letterHolder, letterHolderCases.get(0));
+        if (CREATE.getValue().equals(caseDetails.getState())) {
+            throw new DefendantLinkingException("Claim is not in open state, can not link defendant");
```

Comment:
DefendantLinkingException is mapped to Forbidden errorcode. is that what we want or we can throw badrequestexception? Forbidden means you are not supposed to access that resource.

Reply:
Yes, that's exactly what we want to return 403 in that scenario.

Reply:
403 will be something I try to access someone else case. 
Trying to link myself to open case cannot be 403.

Reply:
So what are you suggesting here to use then?

Reply:
409 Conflict - would be correct. Thats what we use for other business rules validation.
`Indicates that the request could not be processed because of conflict in the current state of the resource` - Conflict explanation

Reply:
As we discussed, leaving that 403.